### PR TITLE
fixed bigiq_license lookup plugin issue

### PR DESCRIPTION
**Pull request by RavinderReddyF5**

### CODE REVIEW

File: ansible_collections/f5networks/f5_modules/plugins/module_utils/bigiq.py [hunk 1]
```diff
@@ -36,15 +36,13 @@ def api(self):
...
             self.provider['server'], self.provider['server_port'], LOGIN
         )
         payload = {
             'username': self.provider['user'],
             'password': self.provider['password'],
         }
-
         # - local is a special provider that is baked into the system and
         #   has no loginReference
         if provider != 'local':
```
Comment (wojtek0806) [inline ansible_collections/f5networks/f5_modules/plugins/module_utils/bigiq.py:48]: code needs to be spaced out for readability

File: ansible_collections/f5networks/f5_modules/plugins/module_utils/bigiq.py [hunk 2]
```diff
@@ -54,16 +52,13 @@ def connect_via_token_auth(self):
...
             url,
             json=payload,
             headers=self.headers
         )
-
         if response.status not in [200]:
             return None, response.content
-
         session.request.headers['X-F5-Auth-Token'] = response.json()['token']['token']
         return session, None
```
Comment (wojtek0806) [inline ansible_collections/f5networks/f5_modules/plugins/module_utils/bigiq.py:63]: space out code for readability

File: ansible_collections/f5networks/f5_modules/plugins/lookup/bigiq_license.py [hunk 3]
```diff
@@ -152,4 +150,5 @@ def run(self, terms, variables=None, **kwargs):
                 keys.append(key)
 
         result = random.choice(keys)
-        return result
+        regKeypool.append(result)
+        return regKeypool
```
Comment (wojtek0806) [inline ansible_collections/f5networks/f5_modules/plugins/lookup/bigiq_license.py:154]: This is supposed to be a single key from a list not the entire list

**Review by wojtek0806 — CHANGES_REQUESTED** [review]

**Comment by wojtek0806** [comment]
rebase squash commits, make sure tests pass and i will rereview :)

**Review by RavinderReddyF5 — COMMENTED** [review]
review comments addressed and fixed build issues

**Pull request merged**